packages/flutter/lib/src/material/tabs.dart

```diff
@@ -96,6 +97,11 @@ class Tab extends StatelessWidget implements PreferredSizeWidget{
   /// [text] or [child] is non-null.
   final EdgeInsetsGeometry iconMargin;
 
+  /// The widget height when there is an icon and text
+  ///
+  /// Useful when the combination of both overflows the default height
```

Comment:
Why make this only specific to one use case? If we are going to expose a `height` setting, it should apply for all configurations.